src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/index.tsx

```diff
@@ -103,7 +103,7 @@ export const CreateNSTreeModal = ({
 
   // Filter based on location
   const { data: groupInfo } = useGroupInfo();
-  const { data: namedLocationsData } = useNamedLocations();
+  const { data: namedLocationsData } = useNamedPathogenDepthLocations();
```

Comment:
I'd like a tiny comment here just explaining that the NSTree creation process restricts the locations you can choose from based on the type of pathogen. Seems like a useful thing to explain here.